backend/debug/available_regs.ml

```diff
@@ -61,18 +61,11 @@ let augment_availability_at_raise avail =
   | Specific_trap (label, _) -> augment_availability_at_exit label avail
 
 let setup_avail_at_raise kind =
```

Comment:
(We could probably delete both
`setup_avail_at_raise` and
`restore_avail_at_raise`, and
then also `avail_at_raise`.)

Reply:
Doing that now